index.html

```diff
@@ -117,37 +117,30 @@ <h2>Network requests</h2>
 
       <p>
       A <dfn data-lt="network requests">network request</dfn> occurs when the
-      user agent must use the network to service a single <dfn data-cite="RFC7230#section-2.1">request</dfn>.
+      user agent must use the network to service a single [=request=].
       </p>
 
       <p>
-      If the user agent can service a <a>request</a> out of a local
-        cache, that <a>request</a> MUST NOT result in a <a>network
-        request</a>.
+      If the user agent can service a [=request=] out of a local cache, that

```

Comment:
I think if we use Fetch we get this for free, but it doesn't seem harmful to leave in.